packages/teams-js/src/private/videoEx.ts

```diff
@@ -84,12 +88,30 @@ export namespace videoEx {
    * @internal
    * Limited to Microsoft-internal use
    */
-  export type VideoFrameCallback = (
+  export type videoBufferHandler = (
```

Comment:
This is the same handler as video.VideoBufferHandler. It would be good to import and use that handler here to avoid having duplicated types/documentation

Reply:
Not exactly.

`type videoExt.VideoBufferHandler = (frame: videoExt.VideoFrame, /* ommited */) => void;`
`type video.VideoBufferHandler = (frame: video.VideoBufferData, /* omitted */) => void;`

Althouth, `videoExt.VideoFrame` extends `video.VideoBufferData`, the function type cannot be unified as function argument types are contravariant.

Reply:
Thanks for the clear explanation, you're right I did miss the extra parameters in the videoEx version.